en/components/grid_filtering.md

```diff
@@ -26,6 +26,9 @@ There's a default filtering strategy provided out of the box, as well as all the
 <igx-column field="Price" filterable="true" dataType="number"></igx-column>
 ```
 
+> [!NOTE]
+> If values of type `string` are used by column of dataType `Date`, the grid won't parse it to `Date` objects and using of filtering conditions won't be possible. If you want to use `string` objects, additional logic should be implemented on application level, in order to parse the values to `Date` object.
```

Comment:
... won't parse it to `Date` objects and using filtering conditions...
or
... won't parse it to `Date` objects and usage of filtering conditions...
I like the first one better, it sounds more natural.

Also, 
... should be implemented on the application level
or
... should be implemented on an application level